packages/entity-browser/src/main.js

```diff
@@ -116,3 +117,34 @@ const initApp = (id, input, events, publicPath) => {
     appFactory.registerAppInRegistry(packageName, initApp)
   }
 })()
+
+class EntityBrowserApp extends React.Component {
+  constructor(props) {
+    super(props)
+    this.app = initApp('id', props)
+  }
+
+  render() {
+    return (
+      <div>{this.app.renderComponent()}</div>
+    )
+  }
+}
+
+EntityBrowserApp.propTypes = {
+  formName: PropTypes.string.isRequired,
+  entityName: PropTypes.string.isRequired,
+  showSearchForm: PropTypes.bool,
+  disableSimpleSearch: PropTypes.bool,
+  formBase: PropTypes.bool,
+  limit: PropTypes.number,
+  preselectedSearchFields: PropTypes.array,
+  searchFilters: PropTypes.array,
+  simpleSearchFields: PropTypes.string,
+  initialKey: PropTypes.string,
+  nullBusinessUnit: PropTypes.bool,
+  showCreateButton: PropTypes.bool,
+  memoryHistory: PropTypes.string
```

Comment:
this must be a boolean (and therefore is wrong in readme). Could you please adjust that.